### PR TITLE
STREAMLINE-709 Import Druid service during the Service pool creation

### PR DESCRIPTION
**Pull request by HeartSaVioR**
This is on top of STREAMLINE-545 (#511). I'll rebase once #511 is merged.
Only commit 10faa71 is for STREAMLINE-709.

Registering Druid service by manual should be easy: just uploading `common.runtime.properties`.

### CODE REVIEW

**Comment by harshach** [comment]
looks good to me. Given that there are high number files changed, I would like reviews from others as well @priyank5485 @hmcl @arunmahadevan take a look.

File: bootstrap/services/druid-bundle.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "DRUID",
+  "registerClass": "com.hortonworks.streamline.streams.cluster.register.impl.DruidServiceRegisterer",
+  "serviceUISpecification": {
+    "fields": [
+      {
+        "uiName": "common.runtime.properties",
```
Comment (arunmahadevan) [inline bootstrap/services/druid-bundle.json]: nit: uiName can be more descriptive like "Druid config file"

Comment (HeartSaVioR) [inline bootstrap/services/druid-bundle.json]: OK. Other uiNames for configuration files are similar, so I'll make the change for all places.

Comment (HeartSaVioR) [inline bootstrap/services/druid-bundle.json]: Addressed.

**Comment by arunmahadevan** [comment]
+1 for commits specific to STREAMLINE-709

**Comment by HeartSaVioR** [comment]
Rebased to apply merged STREAMLINE-545.

**Pull request merged**